rllib/models/torch/misc.py

```diff
@@ -95,11 +95,11 @@ def __init__(self,
                  bias_init=0.0):
         super(SlimFC, self).__init__()
         layers = []
-        linear = nn.Linear(in_size, out_size)
+        self.linear = nn.Linear(in_size, out_size)
```

Comment:
Need to access this in LSTMs to be able to properly create the internal state tensors (on same device as the entire Model).